package.json

```diff
@@ -62,6 +62,7 @@
     "@changesets/cli": "^2.26.0",
     "@cloudflare/vitest-pool-workers": "^0.7.8",
     "@cloudflare/workers-types": "^4.20230307.0",
+    "@hono/eslint-config": "workspace:*",
```

Comment:
Need this here so that `yarn workspaces focus hono-middleware @hono/...` will also install `@hono/eslint-config`